Gemfile

```diff
@@ -23,6 +23,7 @@ end
 group :development, :test do
   gem 'govuk-lint'
   gem 'wraith', '~> 3.1'
+  gem 'timecop', '0.8.0'
```

Comment:
Rails has `travel_to` built in since `4.1` which has feature-parity with `timecop` and doesn't require yet another dependency.